subprojects/language-java/src/integTest/groovy/org/gradle/javadoc/JavadocIntegrationTest.groovy

```diff
@@ -253,10 +258,10 @@ Joe!""")
 
         file("build/tmp/javadoc/javadoc.options").assertContents(containsNormalizedString("""-addStringsOption 'a b c'"""))
 
-        file("build/tmp/javadoc/javadoc.options").assertContents(containsNormalizedString("""-addMultilineMultiValueOption 
-'a' 
--addMultilineMultiValueOption 
-'b' 'c' """))
+        file("build/tmp/javadoc/javadoc.options").assertContents(containsNormalizedString("""-addMultiValueOption 'a' 'b' 'c'"""))
+
+        file("build/tmp/javadoc/javadoc.options").assertContents(containsNormalizedString("""-addMultilineMultiValueOption 'a'
+-addMultilineMultiValueOption 'b' 'c'"""))
```

Comment:
Fixed a minor bug in `MultilineMultiValueJavadocOptionFileOption` described [here](https://github.com/gradle/gradle/issues/6991#issuecomment-427278068)